tests/jsunit/test_utilities.js

```diff
@@ -89,3 +89,54 @@ function checkVariableValues(container, name, type, id) {
   assertEquals(type, variable.type);
   assertEquals(id, variable.getId());
 }
+
+/**
+ * Create a test get_var_block.
+ * Will fail if get_var_block isn't defined.
+ * TODO (fenichel): Rename to createMockVarBlock.
+ * @param {!string} variable_id The id of the variable to reference.
+ * @return {!Blockly.Block} The created block.
+ */
+function createMockBlock(variable_id) {
```

Comment:
Perhaps createMockGetVarBlock(..)
(I know this was just a move and was already in use)